### PR TITLE
{math}[foss/2018b,intel/2018b] Gurobi v9.0.0 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by schiotz**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/6b22d2a928fdce0edb28774563fb97b2 for a full test report.

**Comment by akesandgren** [comment]
Please provide a test report for this since it requires a license.

eb --from-pr=9435 -u -r

**Comment by schiotz** [comment]
Test report by @schiotz
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
slid.fysik.dtu.dk - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/d2c10b5bd03e2cc950db63d7adedfdb1 for a full test report.

**Comment by schiotz** [comment]
Test report by @schiotz
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
svol.fysik.dtu.dk - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/4b20bcbd36878bf6d206df7f57b7857b for a full test report.

**Comment by schiotz** [comment]
@akesandgren Good point.  Done.

File: easybuild/easyconfigs/g/Gurobi/Gurobi-9.0.0-intel-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+homepage = 'https://www.gurobi.com'
+description = """The Gurobi Optimizer is a state-of-the-art solver for mathematical programming.
+The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern 
+architectures and multi-core processors, using the most advanced implementations of the 
+latest algorithms."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+# registration is required
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/Gurobi/Gurobi-9.0.0-intel-2018b-Python-3.6.6.eb]: Comment is slightly misleading. You don't need to register to download (worked for me with --from-pr)

Comment (schiotz) [reply easybuild/easyconfigs/g/Gurobi/Gurobi-9.0.0-intel-2018b-Python-3.6.6.eb]: Oops, that comment has survived from version 7.X where that was the case.  Removed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2414.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ad4b119fa109e2c875c4cc70d215c608 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @schiotz!

**Pull request merged**